pkgs/development/python-modules/py-sr25519-bindings/default.nix

```diff
@@ -11,26 +11,24 @@
 
 buildPythonPackage rec {
   pname = "py-sr25519-bindings";
-  version = "0.2.0";
+  version = "2023-03-15-unstable";
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "polkascan";
     repo = "py-sr25519-bindings";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-Lu3J0+YeQHHKItOZTT24DlQAUJuE9fd+py6Eb46/MSE=";
+    rev = "9127501235bf291d7f14f00ec373d0a5000a32cb";
```

Comment:
This is not a build fix, this is a bump to a unstable revision, for which reason?

Reply:
Sorry, the commit message was wrong. This commits fixes the build in upstream